### PR TITLE
Check blacklist when processing a block #1464

### PR DESCRIPTION
**Pull request by xuecanlong**
done

### CODE REVIEW

File: blockchain/blockvalidator_test.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ package blockchain
 
 import (
 	"context"
+	"github.com/iotexproject/iotex-address/address"
```
Comment (zjshen14) [inline blockchain/blockvalidator_test.go]: let's group imports in the order:

std

third party

first party

Comment (xuecanlong) [reply blockchain/blockvalidator_test.go]: ok, fixed

**Comment by zjshen14** [comment]
@xuecanlong can you check why the ci build failed?

**Comment by xuecanlong** [comment]
ok, already fixed.

**Pull request merged**